classes/autoptimizeConfig.php

```diff
@@ -8,10 +8,22 @@ class autoptimizeConfig {
 	//Singleton: private construct
 	private function __construct() {
 		if( is_admin() ) {
+			
+			// Load custom admin styles
+			add_action( 'admin_enqueue_scripts', array( $this, 'load_custom_wp_admin_style' ) );
+
+			// Load custom admin javascript
+			add_action('admin_enqueue_scripts', array( $this, 'load_custom_wp_admin_javascript' ) );
+			
 			//Add the admin page and settings
 			add_action('admin_menu',array($this,'addmenu'));
 			add_action('admin_init',array($this,'registersettings'));
 
+			// Load admin toolbar feature
+			if(file_exists(AUTOPTIMIZE_PLUGIN_DIR.'classes/autoptimizeToolbar.php')){
```

Comment:
we should be able to safely assume autoptimizeToolbar.php is available. instead you could check for $show here?
`if (current_user_can( 'manage_options' ) || current_user_can( 'edit_others_pages' ) ) ? true : false;)`